1.1.0 Changes

cs-abhishek-shukla opened the pull request:
- Added full screen images
- Rewrote setup, readme, and Usage
- Annotated screenshots
- Added release notes

#### Descriptions:
_Add PR descriptions here_

#### Fix:
_Add fix/changes here_

cs-abhinav-goel commented on README.md:
```diff
@@ -1,6 +1,6 @@
 # Release Information
 
-- **Version**: 1.0.0
+- **Version**: 1.1.0
```
change the minimum FSR version to 7.4.3 or later

cs-abhinav-goel commented on docs/setup.md:
```diff
@@ -1,44 +1,61 @@
 
-4. Click **Configure** on the bottom to begin configuration.
+4. Click **Install** on the bottom to begin installation.
 
```
Add a note, that this widget is only useful when using with CICD SP.
@cs-abhishek-shukla

cs-abhishek-shukla replied:
It is in `readme.md`.
>The **CICD Configuration** widget helps set up source control and streamline the content development lifecycle across multiple FortiSOAR environments designated as Development, Staging, and Production through a configuration wizard.

cs-abhishek-shukla replied:
Added in setup.md

cs-abhinav-goel commented on docs/setup.md:
```diff
@@ -1,44 +1,61 @@
 The configuration settings of the CICD Configuration widget include the following steps to setup CICD configuration:
 
-- Click the **Configure** button to bring up the following wizard screen
+1. Click the **Configure** button to bring up the following wizard screen.
```
@cs-abhishek-shukla : Before this, point we should give the details from where user can click configure button.

cs-abhishek-shukla replied:
Added

cs-abhinav-goel commented on docs/setup.md:
```diff
@@ -1,44 +1,61 @@
+        2. GitLab
 
-    ![Connect Version Control](./res/connect_version_control.png)
+    2. Provide the base URL of the source control.
```
@cs-abhishek-shukla : This option is no more available. Also, update the screenshot for the same

cs-abhishek-shukla replied:
Removed, and updated.

cs-abhinav-goel commented on docs/setup.md:
```diff
@@ -1,44 +1,61 @@
-- Click **Finish** to complete the configuration.
+    3. Click **Next** on lower-right corner to proceed.
+
+        ![Select Source Control](./res/select_source_control.png)
```
@cs-abhishek-shukla : Please take the latest screenshot

cs-abhishek-shukla replied:
Done

cs-abhinav-goel commented on docs/setup.md:
```diff
@@ -1,44 +1,61 @@
+        ![Define Environment](./res/define_environment.png)
 
-- Click **Next** on lower-right corner
+4. **Define Your Source Control**
```
@cs-abhishek-shukla : Take the new screenshot of the wizard. Also, now the heading is "Select a Source Control"

cs-abhishek-shukla replied:
Done

cs-abhinav-goel commented on docs/setup.md:
```diff
@@ -1,44 +1,61 @@
+
+        ![Select Source Control](./res/select_source_control.png)
+
+5. **Connect Version Control**
```
Now the page name is "Configure Source Control" or "Configure the Source Control".
Also take the latest screenshot for the same.
@cs-abhishek-shukla

cs-abhishek-shukla replied:
Done

cs-abhinav-goel commented on docs/setup.md:
```diff
@@ -1,44 +1,61 @@
+
+5. **Connect Version Control**
+
+    1. Specify the selected source control's configuration details.
```
Rather than saying specify, can't we say add a configuration for the selected Source control.

Also,if the user has not added the configuration then they can skip it.
But if they have added the configuration then they must click on next button.

@cs-abhishek-shukla

cs-abhishek-shukla replied:
Done

cs-abhinav-goel commented on docs/setup.md:
```diff
@@ -1,44 +1,61 @@
+
+- Click **Next** to complete the configuration.
 
     ![Finish Configuration](./res/finish_configuration.png)
```
@cs-abhishek-shukla : This screenshot will also change. please connect with Sujit for the same.

cs-abhishek-shukla replied:
Done

cs-abhinav-goel commented on docs/usage.md:
```diff
@@ -9,6 +9,9 @@ The CICD Configuration Wizard widget helps set up source control and streamline
 - `Define Environment` - Helps choose the FortiSOAR environment type, from development and production, as per your requirements 
 
-- `Connect Version Control` - Configures GitHub version control to perform Continous Delivery operations
+- `Define Your Source Control` - Select the source control you want to configure and provide the base url of the source control.
```
This is now "Select Source Control" @cs-abhishek-shukla

cs-abhinav-goel replied:
Also, it no more needs "Base URL" of source control

cs-abhishek-shukla replied:
done

cs-abhinav-goel commented on docs/usage.md:
```diff
@@ -9,6 +9,9 @@ The CICD Configuration Wizard widget helps set up source control and streamline
-- `Connect Version Control` - Configures GitHub version control to perform Continous Delivery operations
+- `Define Your Source Control` - Select the source control you want to configure and provide the base url of the source control.
 
+- `Connect Version Control` - Configures GitHub version control to perform Continous Delivery operations
```
Also, please do the similar changes as per the new build.
@cs-abhishek-shukla

cs-abhishek-shukla replied:
Changed

cs-abhinav-goel reviewed (changes requested):
@cs-abhishek-shukla : Please do the changes.

cs-abhishek-shukla committed:
1.1.0 Changes
- Added full screen images
- Rewrote setup, readme, and Usage
- Updated screenshots
- Added release notes
